### PR TITLE
Feature: APP-2601 - Successful Multisig proposal status

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description

- shows `Approved` for expired signaling Multisig proposals

Task: [APP-2601](https://aragonassociation.atlassian.net/browse/APP-2601)

TODO:
- ~[x] Rebase and set to ready for review after #1143 has been merged~

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2601]: https://aragonassociation.atlassian.net/browse/APP-2601?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Fabricevladimir** [comment]
@emmdim This PR updates the package `@aragon/sdk-client-common` to v. 1.12.0

File: src/hooks/useMappedBreadcrumbs.tsx [hunk 1]
```diff
@@ -23,6 +26,18 @@ type MappedBreadcrumbs = {
...
 
+type ProposalStatusColorMap = {
+  [key in ProposalStatus]: TagColorScheme;
+};
+
+const proposalStatusColorMap: ProposalStatusColorMap = {
+  [ProposalStatus.ACTIVE]: 'info',
+  [ProposalStatus.PENDING]: 'neutral',
+  [ProposalStatus.SUCCEEDED]: 'success',
+  [ProposalStatus.EXECUTED]: 'success',
```
Comment (sepehr2github) [inline src/hooks/useMappedBreadcrumbs.tsx:37]: Is it intentional? Since you used success for both statuses

**Review by sepehr2github — COMMENTED** [review]
Thanks for the changes, Quick question if the proposal threshold met and approved the status should be succeed or active (before proposal end date ofcourse)?

File: src/hooks/useMappedBreadcrumbs.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply src/hooks/useMappedBreadcrumbs.tsx:37]: Yes

**Comment by Fabricevladimir** [comment]
> Thanks for the changes, Quick question if the proposal threshold met and approved the status should be succeed or active (before proposal end date ofcourse)?

It should be active

**Review by sepehr2github — APPROVED** [review]
thanks, LGTM

**Pull request merged**